test/testFortranMPAdvection.f90

```diff
@@ -1,58 +1,259 @@
+module advectionTests
+  contains
+  include "calculateDisplacement.f90"
+
+  subroutine setProcWedges(mpMesh, nCells, comm_size, nEdgesOnCell, verticesOnCell, lonCell)
+    use :: polympo
+    use :: readMPAS
+    use :: iso_c_binding
+    implicit none
+
+    type(c_ptr) :: mpMesh
+    integer :: i, j, k, nCells, comm_size
+    integer, dimension(:), pointer :: owningProc, nEdgesOnCell
+    integer, dimension(:,:), pointer :: verticesOnCell
+    real(kind=MPAS_RKIND), dimension(:), pointer :: lonCell
+    real(kind=MPAS_RKIND) :: normalizedLat, min, max
+    
+    allocate(owningProc(nCells))
+    
+    min = 1000
+    max = -1000
+    do i = 1, nCells
+      if (lonCell(i) < min) min = lonCell(i)
+      if (lonCell(i) > max) max = lonCell(i)
+    end do
+
+    do i = 1, nCells
+      normalizedLat = (lonCell(i) - min) / (max - min) * .99
```

Comment:
What is this 0.99 factor? I do not recall.

Reply:
This code is meant to normalize the longitude so they can be sorted to different mpi ranks. But when the max is equal to the longitude it will go out of bounds because the max valid rank is mpi_size-1. So this line will normalize from 0 to .99

Reply:
Let's open a ticket/issue for now. Something like: clean up parallel advection test.

For example, it is not clear why 0.99 and so 0.999 or 0.9999. So I need to think more on this.